react/bingo/src/bingo/Game.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import GameLogic from './GameLogic'
+
+import './BingoApp.css';
+
+class Game extends React.Component {
```

Comment:
Can extend React.PureComponent as there is no state

Reply:
Done in Bingo